### PR TITLE
Improve data export with the CLI

### PR DESCRIPTION
**Pull request by hemp**
Improve data export with the CLI by making a single command to run versus
the current 6 step process.

`./life-export` will configure defaults, ask the user to authenticate,
and pull data locally as JSON under a directory named `life-data`.

image

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -5,14 +5,19 @@ set -x
...
-(cd target/linux-x86 && zip -q -9 ../lo-linux-x86.zip lo)
 (cd target/windows-x64 && zip -q -9 ../lo-windows-x64.zip lo.exe)
-(cd target/windows-x86 && zip -q -9 ../lo-windows-x86.zip lo.exe)
 (cd target/macos-x64 && zip -q -9 ../lo-macos-x64.zip lo)
+
+node_modules/.bin/pkg --target node12-linux-x64 --output target/linux-x64/life-export life-export.js
+node_modules/.bin/pkg --target node12-win-x64 --output target/windows-x64/life-export.exe life-export.js
+node_modules/.bin/pkg --target node12-macos-x64 --output target/macos-x64/life-export life-export.js
+
+(cd target/linux-x64 && zip -q -9 ../life-export-linux-x64.zip life-export)
```
Comment (anthonyroach) [inline build.sh]: Since life-export requires lo can you add lo to the zip here so they don't have to download and extract both separately?

Comment (hemp) [reply build.sh]: Good call

**Comment by nickmartin90** [comment]
@hemp without having much context...is there any issue hard coding LIFE_PROJECT and client id?

**Comment by hemp** [comment]
> @hemp without having much context...is there any issue hard coding LIFE_PROJECT and client id?

We are good, that project ID used to be in Zendesk as part of the manual steps and is a public identifier.

**Pull request merged**